scripts/SANS/isis_reduction_steps.py

```diff
@@ -4282,7 +4282,7 @@ def execute(self, reducer, workspace):
             Reads the geometry information stored in the workspace
             but doesn't replace values that have been previously set
         """
-        _reducer = reducer  # noqa
+        _ = reducer  # noqa
```

Comment:
I cannot remember why I added it, but I suspect that it was because of either a pylint or flake8 warning (or both), because we have an unused argument.

Since the class is a reduction step it has an `execute(self, reducer, workspace)` method. where the second argument is just not used, but it is part of the contract.